packages/core/src/actors/observable.ts

```diff
@@ -45,9 +45,10 @@ export function fromObservable<T, TInput>(
   const errorEventType = '$$xstate.error';
   const completeEventType = '$$xstate.complete';
 
-  return {
+  // TODO: add event types
+  const logic: ObservableActorLogic<T, TInput> = {
```

Comment:
is there any need to move it to a variable? it's quite nice that we don't have to repeat the declared type here if we just rely on the contextual return type applied to the return statement

Reply:
No need, can revert